152_data_reading_program/sensor_reader.py

```diff
@@ -2,50 +2,81 @@
 # the MCP-2221. This script is influenced by Carter Nelson and
 # Bryan Siepert's scripts written for Adafruit Industries in 2020 and 2021
 # It enhances these scripts by calling the linux export statement directly,
-# By presenting the data much more cleanly
-# by having a discrete function to reset the SCD and calling this function
-# when necessary when the SCD-30 crashes (which happens sometimes)
-# and also by recovering from failure to read from the SCD-30
+# By avoiding any kind of scd.reset() or alternative method which crashes
+# the sensor, and by recovering from failure to read from the SCD-30
 # by trying again next time instead of just crashing altogether.
-# Unfortunately, there is still a problem where the sensor stops working
-# eventually and must be unplugged from the computer and plugged back in
-# again to make it work.
-# @author Greg Ross, version 1, October 29,2021
+# This script has allowed me to continuously run the sensor for over 8 hours
+# as opposed to the one they provide which crashes in a few minutes
+# and the alternate reset method they propose which tends to crash
+# in about 30 minutes.
 
-# According to Carter'script, the following commented line causes
-# prevents an issue with the reset on MCP-2221
-# pylint: disable=protected-access
+# @author Greg Ross, version 2, October 31,2021
 
-import sys # For force quit.
-import os # To do the system call to set export
+# Normal Packages
+import os
+import sys
+import time
+# Special packages
+import busio
+import adafruit_scd30
+
+# export statment is required to import board
 # Set the export statment command with a system call
 # in case it is not already set.
 # If this system call is not made, import board will fail
+# This system call sometimes works on my computer but usually fails
 os.system("export BLINKA_MCP2221=1")
-import time # For sleep function
-import adafruit_scd30 # Adafruit's Library for the SCD-30
-import busio # For I2C
+
+try:
+   os.environ["BLINKA_MCP2221"]
+except KeyError:
+   print(f'Before running this program, type "export BLINKA_MCP2221=1"'
```

Comment:
Without extra space at the end, the output will be `type "export BLINKA_MCP2221=1"in the same` since string literals are concatenated.

Here you could just use `if 'BLINKA_MCP2221' not in os.environ: ...`.